builders/pallets-precompiles/precompiles/referenda.md

```diff
@@ -183,9 +177,7 @@ Referenda Precompile位于以下地址：
 要提交保证金，请执行以下步骤：
```

Comment:
Polkassembly link above this can be changed to https://moonbase.polkassembly.io/opengov
Polkassembly.io links will always direct to the latest gov versions, whereas Polkassembly.network may go to old version gov 1

Reply:
It defaults to opengov for me, but i'll add it just to be sure